### DISCUSSION TITLE
关于需要使用数据包加载的部分汉化处理

### DISCUSSION THREAD

**Discussion by dovisutu** [CFPAOrg/Minecraft-Mod-Language-Package#957] [Q&A] [answered]
目前，存在一些 mod 的手册等，如*1.16的patchouli***无法使用资源包加载**，体现为在源文件中存储于与 *`assets/`* 所**平行**的 __*`data/`*__ 中；这些汉化需要 _按存档计的**数据包**_ 才可以加载（不过*未实测*，谁可以去实测一下）
（*但1.12的patchouli实测可以使用资源包加载*）

这样是我们目前**无法处理**的。目前对于这些东西，我有一些想法：
 - 修改i18nupdatemod（再一次）以在下载时同样放置每存档的数据包并自动挂载
 - 对目前的仓库不可避免地进行一定修改：
   - 或许可以通过在某处存储一项文件的归属：数据包//资源包//二者（为什么要有这个？）//皆不（为什么要有这个？），并且更改packer使其基于此归属表打包
     - 这样也许可以往这里放入更多东西，例如连spider读取的mod信息也可以全部丢在这里（~只不过这样子文件就很大了~
   - 或许可以分为两个仓库，但是共用同一个weblate服务器？
     - 不过在手册上wl之前或许可以先搁置

> 有些地方加了空格，因为markdown的识别有点奇怪...
> 此外，为什么在手机端查看似乎没有粗体...

**Comment by kappa-maintainer** [comment, marked as answer]
现时点已经通过mixin单独解决了patchouli的问题
https://github.com/CFPAOrg/I18nUpdateMod2/issues/3
https://www.curseforge.com/minecraft/mc-mods/prp

**Reply by AnzhiZhang** [reply]
依然有个小问题，不用 I18nUpdateMod 无法解决，需要同时安装资源包和 Mod。

**Reply by kappa-maintainer** [reply]
你可以尝试让帕秋莉官方改变加载流程
目前1.16和1.18都存在这个问题

**Reply by AnzhiZhang** [reply]
同样使用Mod加载汉化即可，但仍需加载Mod，举例：[ShaBaiTianCN/ProductiveBeesChineseGuidebook](https://github.com/ShaBaiTianCN/ProductiveBeesChineseGuidebook)

**Reply by kappa-maintainer** [reply]
我就是说不需要额外模组的办法
只能让官方改加载流程

**Reply by AnzhiZhang** [reply]
我就是说这个解决问题的方式依然有问题
只能让官方改

**Reply by kappa-maintainer** [reply]
那官方不改的话塞进更新mod已经是最好的结果了
不过我当初不在Vazkii的DC，不清楚具体情况
你可以看看到底能不能合进去

**Reply by AnzhiZhang** [reply]
官方的[更新计划](https://vazkiimods.github.io/Patchouli/docs/upgrading/upgrade-guide-117/)说 1.18 要改为资源包加载，这应该就可以解决翻译问题，只能期待更新并完善 1.16，1.18版本的模组更新了，不知道是否已经实现此功能

**Reply by kappa-maintainer** [reply]
我确定地说没完全解决，1.16似乎处于遗弃状态，1.18则是数据包模式下不行
至于官方态度如何我并不清楚，我只是个写mixin的

**Comment by baka-gourd** [comment]
1.17的patchouli已經改過了

**Reply by kappa-maintainer** [reply]
[官方文档](https://vazkiimods.github.io/Patchouli/docs/upgrading/upgrade-guide-117)

**Reply by dovisutu** [reply]
confirmed
